packages/boba/contracts/contracts/bridges/L2NFTBridge.sol

```diff
@@ -46,6 +46,7 @@ contract L2NFTBridge is iL2NFTBridge, CrossDomainEnabled, ERC721Holder, Reentran
     address public owner;
     address public l1NFTBridge;
     uint256 public extraGasRelay;
+    uint32 public DEFAULT_FINALIZE_WITHDRAWAL_L1_GAS;
```

Comment:
aha yes, only noticed now, i think some of these gas variables in the LPs too are still all-caps, even though they are not constants anymore. maybe we can change them to 'mixedCase' later, if you want, can change the ones in both bridges already. helps stay under the convention 😄 
(the other one is SETTLEMENT_L2_GAS)